tests/test_indices.py

```diff
@@ -269,6 +269,13 @@ def test_static_daily_temperature_range(self):
         dtr = xci.daily_temperature_range(tasmax, tasmin, freq="YS")
         np.testing.assert_array_equal([dtr.mean()], [20])
 
+    def test_static_variable_daily_temperature_range(self):
```

Comment:
I'm not sure I understand the logic of that test. It does not seem very discriminating. I suggest using a synthetic input array that you know exactly what answer it should yield and compare against that.

Reply:
All I'm looking for is that vDTR is between 0 and 10. Not very robust. I'll see if I can throw together a test now to catch edge cases. Not sure what else to test for.